src/data-pipeline/utils/utils-role.ts

```diff
@@ -124,6 +124,7 @@ export class RoleUtil {
           'glue:CreatePartition',
           'glue:BatchCreatePartition',
           'glue:GetUserDefinedFunctions',
+          'glue:TagResource',
```

Comment:
Do we create the database with tags? Let us add 'removeTag' and 'updateTag' if they are also available.

Reply:
![image](https://github.com/awslabs/clickstream-analytics-on-aws/assets/23469402/1c2340f2-a45c-48d5-b516-30e143dd1e37)
added `glue:UntagResource`